benchmark/es/foreach-bench.js

```diff
@@ -23,7 +23,9 @@ function useForOf(n, items) {
   bench.start();
   for (let i = 0; i < n; i++) {
     // eslint-disable-next-line no-unused-vars
```

Comment:
nit: re-use the eslint disable comment instead
```suggestion
    // eslint-disable-next-line no-unused-vars no-empty
```